regolith/builders/preslistbuilder.py

```diff
@@ -166,13 +166,23 @@ def latex(self):
                         2: "$^\mathrm{nd}$",
                         3: "$^\mathrm{rd}$",
                     }.get(pres["begin_day"] % 10, "$^\mathrm{th}$")
+                    for day in ['begin_day', 'end_day']:
+                        if pres.get(day,None):
+                            if 10 < pres[day] < 20:
```

Comment:
This is pretty slick! Awesome! (we might consider making this into a function and putting it into tools?)